src/org/opendatakit/briefcase/ui/export/components/ExportFormsTableView.java

```diff
@@ -50,15 +49,12 @@
 
     addMouseListener(new MouseAdapterBuilder().onClick(this::relayClickToButton).build());
 
-    Dimension formNameDims = getHeaderDimension(HEADERS[FORM_NAME_COL]);
-    Dimension exportStatusDims = getHeaderDimension(HEADERS[EXPORT_STATUS_COL]);
-    Dimension lastExportDims = getHeaderDimension(HEADERS[LAST_EXPORT_COL]);
+    Dimension formNameDims = TableCustomizer.getHeaderDimension(HEADERS[FORM_NAME_COL], this);
```

Comment:
I'd suggest adding a static import of all these methods in TableCustomizer to make the code less verbose.

Reply:
Improved.